app/api/__init__.py

```diff
@@ -15,23 +15,38 @@
 api.route(UserDetail, 'user_detail', '/users/<int:id>')
 
 # tickets
-api.route(AllTicketList, 'all_ticket_list', '/tickets', '/events/<int:event_id>/tickets')
-api.route(TicketDetail, 'ticket_detail', '/tickets/<int:id>')
+api.route(AllTicketList, 'all_ticket_list', '/tickets', '/events/<int:id>/tickets')
```

Comment:
In the tickets.py if you see, the logic is implemented using id and not event_id as the key. And the name here doesn't need to be event_id. Because there can be only one kind of scenario. Where there are different url endpoints possible with different ids, then we need to specify it explicitly. For example, line 19